### PR TITLE
restyling enterprise documentation 

### PR DESCRIPTION
**Pull request by tonyco97**
Use rtd sphinx theme because have a better layout on mobiles and also browsers

Previews:
- https://nethservice.nethesis.it/light/
- https://nethservice.nethesis.it/dark

### CODE REVIEW

**Review by gsanchietti — APPROVED** [review]
It seems good to me.
Let's wait @DavidePrincipi  approval

**Review by gsanchietti — CHANGES_REQUESTED** [review]
I've just tried a build: the new logo is missing.

Extract from build log:
```
WARNING: logo file 'nsent/_static/nethesis_enterprise_yellow.png' does not exist
```

File: administrator-manual/en/conf.py [hunk 1]
```diff
@@ -115,19 +110,14 @@
...
-        'bootswatch_theme': "cerulean",
         'nosidebar': "1",
+        'collapse_navigation': True,
+        'navigation_depth': -1,
+        'logo_only': True,
+        'style_nav_header_background': '#343131',
     }
 
 elif tags.has('nscom'):
-    templates_path = ['nscom/_templates']
```
Comment (DavidePrincipi) [inline administrator-manual/en/conf.py:130]: Why was this line removed?

File: administrator-manual/en/nsent/_static/nethesis.css [hunk 2]
```diff
@@ -1,43 +1,51 @@
...
-    background-color: #efefef;
-    font-size: 15px;
+    color:  	#5B5B5B;
+    background-color: white;
+    font-size: 17px;
     line-height: 24px;
 }
 .container {
-    background-color: #fff;
+    background-color:  	#B0C4DE;
```
Comment (DavidePrincipi) [inline administrator-manual/en/nsent/_static/nethesis.css:9]: Do not insert TAB characters. For instance here (and also below)

```suggestion
    background-color: #B0C4DE;
```

File: administrator-manual/en/nsent/_static/nethesis.css [hunk 3]
```diff
@@ -309,4 +343,38 @@ footer .pull-right a:hover {
...
+}
+.rst-content .sidebar .sidebar-title {
+    border-radius: 15px !important;
+   }
+.wy-nav-top {
+    background:#343131 !important;
+}
+.wy-menu-vertical li.current {
+    background:#c9c9c9 !important;
+}
```
Comment (DavidePrincipi) [inline administrator-manual/en/nsent/_static/nethesis.css:380]: Text files usually end with a NEWLINE. 

I guess you have to tweak your editor configuration for white space management ;)

**Review by DavidePrincipi — COMMENTED** [review]
Thank you for the PR @tonyco97 it seems good. Some secondary things to note down:

- Start commit and PR titles with a capital letter
- Do not use TAB characters: indent with spaces
- Insert a trailing NEWLINE at the end of the file

See the individual notes for details.

File: administrator-manual/en/nsent/_static/nethesis.css (hunk 3 above)
Comment (tonyco97) [reply administrator-manual/en/nsent/_static/nethesis.css:380]: Thanks for all the review, i'll charge in few moments a new commit with the fix

File: administrator-manual/en/conf.py (hunk 1 above)
Comment (gsanchietti) [reply administrator-manual/en/conf.py]: There are no tenplates for nscom, so I proposed to remove it.
I'm ok even if we want keep it.

**Review by DavidePrincipi — APPROVED** [review]

**Review by gsanchietti — APPROVED** [review]
